aea/mail/base.py

```diff
@@ -482,104 +478,67 @@ def connection_status(self) -> ConnectionStatus:
         """Get the connection status."""
         return self._connection_status
 
-    def connect(self) -> None:
+    async def connect(self) -> None:
         """Connect the multiplexer."""
-        with self._lock:
+        self._out_queue = asyncio.Queue()
```

Comment:
Where did we previously initialise the outqueue?

Reply:
it's better to set queue whet event loop is already set.